context.go

```diff
@@ -151,17 +151,21 @@ func (c *Context) setDefaultTags() error {
 	return nil
 }
 
-// Stats gets stats for a context as a string
-func (c *Context) Stats() (string, error) {
+// Stats gets stats for a context as json bytes
+func (c *Context) Stats() ([]byte, error) {
 	var stats *C.char
 	if ret := C.tiledb_ctx_get_stats(c.tiledbContext, &stats); ret != C.TILEDB_OK {
-		return "", fmt.Errorf("Error getting stats from context: %s", c.LastError())
+		return nil, fmt.Errorf("Error getting stats from context: %s", c.LastError())
```

Comment:
feel free to take or leave this idea, but you might consider returning `b'null'` (or `b'{}'`) here so that even in error cases the returned value is valid JSON.

Reply:
let's proceed with this as is for now. I'm not opposed to it, but in this case I feel like if we return an error from this function the expectation would be that the json result does not contain any valid information (jmo)